### PR TITLE
Add pop-up to portfolio

### PR DESCRIPTION
**Pull request by Luchy-B**
- Add pop-up design to portfolio
- Add Functionality using JS to pop-up

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -151,54 +155,32 @@ body {
...
-.content-List .canopy {
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  padding-top: 1rem;
   color: #344563;
-  font-weight: 600;
 }
 
 .btn-1 {
```
Comment (huseyinbiyik) [inline styles.css:186]: Great job, the buttons are working correctly 👌 But let's add some space to the bottom of the button. 
Screenshot 2023-02-03 at 11 27 52 PM
Screenshot 2023-02-03 at 11 28 12 PM

File: index.html [hunk 2]
```diff
@@ -297,8 +296,59 @@ <h2>Contact me</h2>
...
+          <p id="modalDes" >
+          </p>
+        <div class="modal-info">
+          <ul id="modalLang" >
+            <!-- <li>html</li>
+            <li>css</li>
+            <li>Javascript</li> -->
+          </ul>
+
+         <hr>
```
Comment (huseyinbiyik) [inline index.html:207]: Please change the color of the hr element to match the figma design 👨‍🎨
Screenshot 2023-02-03 at 11 26 04 PM

File: index.html [hunk 3]
```diff
@@ -297,8 +296,59 @@ <h2>Contact me</h2>
...
+    <section class="modalContainer" id="modalContainers">
+      <div class="modalHeader">
+          <div class="modalNav">
+            <h3 class="card-title" id="modalTitle"></h3>
+            <div class="modalNavLi" id="close-btn">&times;
+
+            </div>
+          </div>
+          <ul class="canopy" id="canopy">
+            <!-- <li class="canopyLi1">CANOPY</li>
```
Comment (huseyinbiyik) [inline index.html]: Kindly don't leave any uncommented code in your codebase. In this way, you can keep your codebase clean, organized and professional 👍 If you need these codes later you can travel through your history with git. 📝 Please don't forget to remove the uncommented code in index.html as well 👍

File: styles.css [hunk 4]
```diff
@@ -421,10 +403,174 @@ body {
...
 }
 
+.alert-Msg {
+  font-size: 12px;
+  font-weight: 600;
+  color: #fff;
+  padding-top: 12px;
+}
+
+/* Modal Design */
```
Comment (huseyinbiyik) [inline styles.css:418]: -Excellent job with the backdrop blur 👨‍🎨 But let's fix left and right padding of the image like on the template.
- Please make sure you don't stretch the image, it is a very bad practice for the images. What you can do is, you can have a vertical overflow and user can go down to read the bottom section of the modal.

Screenshot 2023-02-03 at 11 21 53 PM
Screenshot 2023-02-03 at 11 22 28 PM

File: styles.css [hunk 5]
```diff
@@ -421,10 +403,174 @@ body {
...
+
+.modalFooter ul {
+  display: flex;
+  list-style: none;
+  gap: 8px;
+  color: #6070ff;
+  margin: 10px 24px 15px 0;
+}
+
+hr {
```
Comment (huseyinbiyik) [inline styles.css:533]: Good job adding separator on mobile version. 👍 But let's add separator also for the desktop version like on the Figma template.
Screenshot 2023-02-03 at 11 19 57 PM

File: styles.css [hunk 6]
```diff
@@ -421,10 +403,174 @@ body {
...
+  margin-bottom: 10px;
+}
+
+.modal-button {
```
Comment (huseyinbiyik) [inline styles.css:540]: Kindly make sure your button matches with Figma design. Here, buttons have different sizes, icon and text is not aligned well and not enough between text and icon.👍
Screenshot 2023-02-03 at 11 20 41 PM
Screenshot 2023-02-03 at 11 21 10 PM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi team,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Almost there

## Highlights
- Projects and popups created dynamically 👌
- Linter set up well and checks pass 🟢
- Github flow followed 👌

## Required Changes ♻️
- You did a great job in general. But let's fix this bug occurs on mobile version. When you open the mobile version, a broken popup and nav opens automatically. Kindly fix that 🦾
Screen Recording 2023-02-03 at 11 25 06 PM

_Check the comments under the review._

## Optional
- Good README. But let's add your logo file or remove it. Currently, it looks a bit unprofessional.
Screenshot 2023-02-03 at 11 56 54 PM


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 7]
```diff
@@ -297,8 +182,52 @@ <h2>Contact me</h2>
...
+          </ul>
+          <div class=" modalImg" id="modal-Img">
+
+          </div>        
+        <div class="modalFooter">
+          <p id="modalDes" >
+          </p>
+        <div class="modal-info">
+          <ul id="modalLang" >
+          </ul>
```
Comment (Shedrack-Sunday) [inline index.html:205]: - [ ]  Also as suggested by the previous reviewer and rightly so, I noticed you did make changes, but the buttons for your live and source links should display on the first render. presently, the pop-up renders but without the button on the screen on the mobile view. This is in line with best practices for web accessibility and gives the best user experience 👏

- Make sure the other space changes recommended by the previous reviewer are taken note of when you fix the buttons.

# Your view

image

# Expected figma render:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @Luchy-B       👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯 



Highlights
✅All linters are checked
✅ Descriptive commit messages 
  
Required Changes ♻️

- [x] You did make efforts in the  Project and you really need a great job. However, there seem to be a few adjustments needed on the mobile view pop-up to make your whole code stand out and look good. 👍 

- [x] The Reason for taking web accessibility seriously onscreen sizes is because it ensures that users with varying devices, including smaller screens such as smartphones, are able to access and use a website effectively. This includes individuals with disabilities who may use assistive technologies such as screen readers or magnifiers to access the web. By ensuring that websites are accessible on smaller viewports, developers can help make the web more inclusive for all users.  Kindly fix the margin on the right of the screen view. good job so far 👏 


_check the comment section for a detailed review_

Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application.


Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.


Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: styles.css [hunk 8]
```diff
@@ -421,10 +403,180 @@ body {
...
+  padding: 8px 20px 20px 20px;
+  background-color: #fff;
+  width: 100%;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  position: relative;
+  height: 100%;
+  border-radius: 1rem;
+}
```
Comment (didierganthier) [inline styles.css:464]: Great job 🎉 addressing some of the changes requested by the previous reviewers. You're almost there 😉 but I noticed that while implementing the changes certain things got introduced. Your popup works and looks very well on mobile but the `border-radius` on the right side isn't quite there for the Desktop. See below 👇🏽 

Screenshot 2023-02-04 at 11 21 18

**Review by didierganthier — CHANGES_REQUESTED** [review]
Hi Team 👥 ,

🎉 Great job on addressing the feedback from the previous review! You've made some solid progress in implementing the requested changes. However, there are still a few areas that need some attention before we can move on to the next project.



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Required Changes 🛠
- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

**Review by huseyinbiyik — APPROVED** [review]
Hi team,

## ✔️ Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text



## Highlights
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- Commit messages are meaningful and written in imperative tense 📝
- Great implementation of the requested changes of previous reviewers 🕸️

## Optional suggestions
- You can move your license text to a license file instead of keeping it on your README directly.
Screenshot 2023-02-05 at 1 43 14 AM

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**